frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.scss

```diff
@@ -1,5 +1,6 @@
 .review-section {
```

Comment:
I noticed the year values on the valuation section of buildings are a bit misaligned now. Do these changes cover that? If not all good I'll change it - just don't want to double up. 
![image](https://user-images.githubusercontent.com/15724124/101417984-a68bc500-38a1-11eb-8d24-5cb1a521eccb.png)

Reply:
looks like it
![image](https://user-images.githubusercontent.com/41091511/101428457-9847a400-38b5-11eb-9cf3-5b54859a1c60.png)